lib/elixlsx/xml_templates.ex

```diff
@@ -195,10 +198,15 @@ defmodule Elixlsx.XMLTemplates do
 
           case content_type do
             :formula ->
+              if not is_nil(content_opts[:value]) do
+                value = "<v>#{content_opts[:value]}</v>"
```

Comment:
value should have a default I believe (as in `value = if ... :do ... :else ""` etc)